app/views/grandparents/new.html.erb

```diff
@@ -9,3 +9,35 @@
       <% end %>
   </div>
 </div>
+
+
+<div class="container">
+    <div class="row justify-content-center">
```

Comment:
This is cool, we'll just have to move it to the actual show page once the devise is set up.